dist/angular-ui-tree.js

```diff
@@ -853,7 +862,7 @@
               //Creating drag element to represent node.
               dragElm = angular.element($window.document.createElement(scope.$parentNodesScope.$element.prop('tagName')))
                   .addClass(scope.$parentNodesScope.$element.attr('class')).addClass(config.dragClass);
-              dragElm.css('width', UiTreeHelper.width(element) + 'px');
+              dragElm.css('width', UiTreeHelper.width(element) + scope.dragElementExtraWidth + 'px');
```

Comment:
to make this code easier to read could you break "UiTreeHelper.width(element) + scope.dragElementExtraWidth" out into a separate variable so it's obvious that the value should be a number and then on this line that variable can be concatenated with the 'px'.
Since the code is evaluated from left to right it won't make a difference but it would be easier to read.